docs/csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md

```diff
@@ -30,93 +30,95 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # Command-line Building With csc.exe
```

Comment:
Done, but we should run a script that Shayne has sent me to fix this across the board